ClassicLibrary.c

```diff
@@ -4,6 +4,44 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // ClassicLibrary.c
-// Program on branch main.
+// Program on branch A5.
 // Program that contains the definiton of functions defined in Header.h.
-// ------------------------------------------------------------------------------------------------
\ No newline at end of file
+// ------------------------------------------------------------------------------------------------
+#include "Header.h"
+
+void Swap (int* a, int* b) {
+   int k = *a;
+   *a = *b;
+   *b = k;
+}
+
+void Heapify (int arr[], int length, int head) {

```

Comment:
It would be helpful to have a description here of what your heap sort does and how you're implementing it. "Developer notes" so to say.